extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/indexing-configuration.xml

```diff
@@ -38,7 +38,7 @@
           <property name="type" value="profile"/>
           <property name="enable" value="${exo.unified-search.connector.people.enable:true}"/>
           <property name="titleField" value="name"/>
-          <property name="searchFields" value="name,firstName,lastName,position,skills"/>
+          <property name="searchFields" value="name,firstName,lastName,position,skills,aboutMe"/>
```

Comment:
Don't we need an upgrade plugin to reindex old profiles ?
CC @ahamdi

Reply:
it's already handled by the ProfileIndexingServiceConnector , it' updating the indexes of the old profiles with the old mapping 
cc @plamarque